upstash/redis/database/types.go

```diff
@@ -32,9 +32,6 @@ type Database struct {
 type CreateDatabaseRequest struct {
 	Region        string   `json:"region"`
 	DatabaseName  string   `json:"database_name"`
-	Tls           bool     `json:"tls"`
-	Consistent    bool     `json:"consistent"`
-	MultiZone     bool     `json:"multizone"`
```

Comment:
Not sure about removing these. Someone might still be using these, even though only defaults can be given. If you have tested and don't see any problem with it, then we can release with a minor version, rather than a patch:

see: https://developer.hashicorp.com/terraform/plugin/best-practices/versioning

Reply:
Please also see:
https://developer.hashicorp.com/terraform/plugin/framework/deprecations#provider-attribute-removal